roles/pulp_resource_manager/templates/pulpcore-resource-manager.service.j2

```diff
@@ -14,19 +14,23 @@ User={{ pulp_user }}
 Group={{ pulp_group }}
 WorkingDirectory=/var/run/pulpcore-resource-manager/
 RuntimeDirectory=pulpcore-resource-manager
+{% if __pulp_version is version('3.13', '>=') -%}
+ExecStart={{ __pulp_daemons_dir }}/pulpcore-worker --resource-manager
+{%- else -%}
 ExecStart={{ __pulp_daemons_dir }}/rq worker \
           -w pulpcore.tasking.worker.PulpWorker -n resource-manager \
           --pid=/var/run/pulpcore-resource-manager/resource-manager.pid \
           -c 'pulpcore.rqconfig' \
           --disable-job-desc-logging
+{%- endif %}
 
 # Enables tracking of process resource consumption at a unit & cgroup level.
 CPUAccounting=yes
 MemoryAccounting=yes
 
 # This provides reconnect support for PostgreSQL and Redis. Without reconnect support, if either
 # is not available at startup or becomes disconnected, this process will die and not respawn.
-Restart=always
+Restart=on-failure
```

Comment:
I'm not quite sure about this one, but that way, the worker itself can signal systemd not to respawn the process infinitely if it is exits cleanly.

Reply:
That is unfortunate. But we then need to solve the busy restart elsewhere.

Reply:
What is the busy restart problem?

Reply:
When the resource manager terminates (because it is not used), systemd will respawn it in an infinite loop.
I am always experiencing this in the dev env, when a syntaxerror is preventing the workers to start up properly.

Reply:
In this case i need to find a way for the unused resource manager to sleep indefinitely.

Reply:
Yes I think having the resource manager sleep indefinitely would be the thing to do. +1

Reply:
It is that way now.